src/random.js

```diff
@@ -0,0 +1,26 @@
+const faker = require('faker')
+const _random = require('lodash/random')
+
+const generators = [
```

Comment:
I don't like the concept of keeping track of the generators in 2 places. Inside `random()`, you can just use `Object.keys(random)` in order to get the list of generators assigned on `random`. That'll allow you to remove this list altogether.